catalogs/auc.yaml

```diff
@@ -1,11 +1,12 @@
 metadata:
   version: 1
+  data_path: auc
 sources:
   al_musawwar:
     description: "Al-Musawwar Magazine Collection"
     driver: iiif_json
     metadata:
-      data_path: auc/al_musawwar
+      data_path: auc/iiif/al_musawwar
```

Comment:
I don't know if we need the `iiif` directory in AUC. We did at one point because ContentDM offered some collections via iiif and some by OAI. If all collections are now available via iiif we can get rid of this directory. If some are only available via oai, then we would want sub directories for OAI, IIIF

Reply:
Same as above, this is for the dlme-transform side more specifically (https://github.com/sul-dlss/dlme-transform/blob/main/config/metadata_mapping.json#L203) so I think this change has to be in place for now. The question of `iiif` vs `oai` is likely separate and should be addressed in tickets related to that, which is good to know. I'll write those tickets up today, but I think this should go in as is for now.